test/seed.test.js

```diff
@@ -2,11 +2,6 @@
 const seedFunctions = require('../database/seedFunctions.js');
```

Comment:
I think this pull request is full of solid work which took lots of time and makes significant improvements to both the seeding script and management of databases. I would recommend at some point circling back to write a couple of tests though.

Reply:
I do have to make tests for sure, and the idea of making children classes is super appealing, makes the code look less messy. The initial appeal of making one class that can talk to all databases was so that creating the class would be unified too. i.e. `new DBManager('cassandra')` rather than `new CassandraManager()`. I think I can have it both ways though, now that I think about it, and that's pretty exciting. I'm gonna merge and then put some work into separating them.